src/Microsoft.IdentityModel.Tokens.Saml/Saml/ClaimProperties.cs

```diff
@@ -0,0 +1,49 @@
+//------------------------------------------------------------------------------
```

Comment:
It looks like you picked up on changes to PR: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/659

Can you scope this PR to just your changes.